docs/source/tutorial_mor_with_anns.md

```diff
@@ -417,7 +417,7 @@ time step to the next. Therefore, these networks implement an internal memory th
 information over time. Furthermore, for each element of the input sequence, the same neural
 network is applied.
 
-In the {class}`~pymor.models.neural_network.NeuralNetworkLSTMInstationaryModel` and the
+In the {class}`!~pymor.models.neural_network.NeuralNetworkLSTMInstationaryModel` and the
```

Comment:
Does not exist. Maybe @HenKlei can rephrase?

Reply:
Done.